src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -60,6 +57,12 @@
     Text bracketCloseText;
     Text plusText;
 
+    private static final String APPEND_TO_WATCH_LIST_FORMAT = "%s;%s#%d";
+    private static final String CREATE_WATCH_LIST_FORMAT = "id:%s#%d";
```

Comment:
Ah, I see. Okay, guess we can do this for now, at the risk of things going wrong if the user edits the filter.

I think the long-term solution would be to keep qualified ids around past the parsing phrase and combine them into a conjunction only upon evaluation.